packages/beacon-node/test/unit/util/kzg.test.ts

```diff
@@ -72,26 +72,78 @@ describe("KZG", () => {
       }
     }
   });
-});
 
-function transactionForKzgCommitment(kzgCommitment: deneb.KZGCommitment): bellatrix.Transaction {
-  // Just use versionedHash as the transaction encoding to mock newPayloadV3 verification
-  // prefixed with BLOB_TX_TYPE
-  const transaction = new Uint8Array(33);
-  const versionedHash = kzgCommitmentToVersionedHash(kzgCommitment);
-  transaction[0] = BLOB_TX_TYPE;
-  transaction.set(versionedHash, 1);
-  return transaction;
-}
+  it("DataColumnSidecars", async () => {
+    const config = createChainForkConfig({
+      ALTAIR_FORK_EPOCH: 0,
+      BELLATRIX_FORK_EPOCH: 0,
+      CAPELLA_FORK_EPOCH: 0,
+      DENEB_FORK_EPOCH: 0,
+      ELECTRA_FORK_EPOCH: 0,
+      FULU_FORK_EPOCH: 0,
+    });
+    const signedBeaconBlock = ssz.fulu.SignedBeaconBlock.defaultValue();
+    const mocks = getBlobCellAndProofs();
+    const blobs = mocks.map(({blob}) => blob);
+    console.log(blobs);
```

Comment:
no console.log in the test

also rename the blob variables (holdover from copy pasta)